lib/hasher/src/CRC64NVMEStrategy.cpp

```diff
@@ -0,0 +1,71 @@
+#include "irods/CRC64NVMEStrategy.hpp"
+
+#include "irods/base64.hpp"
+#include "irods/checksum.h"
+#include "irods/rodsErrorTable.h"
+
+#include <boost/crc.hpp>
+
+#include <array>
+#include <cstring>
```

Comment:
```suggestion
#include "irods/base64.hpp"
#include "irods/checksum.h"
#include "irods/irods_error.hpp"
#include "irods/rodsErrorTable.h"

#include <boost/any.hpp>
#include <boost/crc.hpp>

#include <array>
#include <cstdint>
#include <cstring>
```

Reply:
This contradicts https://github.com/irods/irods/pull/8701#discussion_r2479225015.

I added the `cstdint` include.

@korydraughn - thoughts